opencti-platform/opencti-front/src/private/components/riskAssessment/risks/remediations/RequiredResourcePopover.js

```diff
@@ -231,6 +245,7 @@ class RequiredResourcePopover extends Component {
       requiredResourceData,
       data,
     } = this.props;
+    const types = ["Party", "Component", "User", "Location", "Inventory-Item", "Party"];
```

Comment:
The values for the "Type" box in the UI should be populated by performing an GraphQL introspection query on the SubjectType.  The query is shown below.  As was the case with Assets, the description field contains what should be shown to the user, where the name contains the value that is to be used in other GraphQL calls.

query SubjectTypeIntrospection {
  __type(name: "SubjectType") {
    name
    enumValues {
      name
      description
    }
  }
}